rfc/2022/2022-07-14_Platform_Security_001_Service_Account_Guidance.md

```diff
@@ -0,0 +1,112 @@
+
+Recommended naming convention:  svc-${PRODUCT}-${TEAM}-${USAGE}
+
+**Examples:**
+
+Good: svc-platform-infra-nightly_cron_job
+
+Bad:  jwilson, service_account, admin
+
+*Required Approval*
+
+The Account Manager is responsible for initially requesting approval (link to request TBD) for the creation and usage of a service account.  Each service account should have a valid authorization approved by the ISO or designee (OCTO-DE VA employee).  In the case of an Account Manager leaving, the replacement Account Manager would need to submit a request for approval by the ISO or designee (OCTO-DE VA employee) designating themselves as the new Account Manager.
+
+*Authenticator Management*
+
+The Account Manager is responsible for the creating, maintaining, safeguarding, and rotating of the service account authenticator. 
+
+In the case of passwords, the Account Manager must ensure that the password for the service account meets minimum password requirements for length and complexity as dictated by VA policy.  Per IA-5(1), the VA requires a case sensitive 12-character password comprised of upper-case letters, lower case letters, numbers and special characters, including at least one of each category.
+
+Authenticators should never be stored in personal password managers.  It is required that authenticators be stored in Parameter Store or an equivalent solution.
+
+The Account Manager can grant (or revoke) access to the authenticator at their discretion amongst their team as necessary for those with a legitimate need to know.  Per IA-5, VA policy requires that service account authenticators be rotated at a minimum of every 3 years, or upon any member of the team or group with access to it leaving or no longer requiring access to the authenticator.  The Account Manager is responsible for providing access to the newly rotated authenticator as needed and ensuring that any application, service, or other process that rely on its use is updated.  
+
+*Intended Usage*
+
+The Account Manager is responsible for ensuring the service account is only used for the intended purpose as stated in the initial request for authorization.  Service accounts shall not be used interactively by an individual or used beyond their intended scope.  If a need arises to use a service account interactively for troubleshooting, approval from the ISO or designee (OCTO-DE VA employee) must be obtained and Platform Security should be CC’ed on the request.  If a need arises to either extend the intended usage of the service account beyond the initial scope, or reduce the scope of the intended usage, a new request documenting the change in intended usage must be submitted and approved by the ISO or designee (OCTO-DE VA employee).
+
+*Documentation and Tracking*
+
+The Account Manager is responsible for documenting, at a minimum, the following information for service accounts they manage:
```

Comment:
Where does this information get documented?

Reply:
It doesn't exist now, but ultimately would need to be maintained in one location by platform security team, likely in in a confluence page.

Reply:
Ok so at this time an Account Manager would be blocked as there is no guidance for where to document this info?

Reply:
Right, it doesn't exist yet.  We're looking now at keeping the list either in VA sharepoint or in a md file in team sensitive repo.